activity/activity.info

```diff
@@ -1,7 +1,7 @@
 [Activity]
 name = JAMClock
 activity_version = 2
-service_name = org.laptop.JAMClock
+bundle_id = org.sugarlabs.JAMClock
 icon = JAMClock
 show_launcher = yes
-class = JAMClock.JAMClock
+exec = sugar-activity3 JAMClock.JAMClock
```

Comment:
Why change to Python 3 wrapper when porting to GTK 3?  Are you also porting to Python 3?  If so, please rename the pull request and change the commit messages.

Reply:
I was planning to port to python 3 first but then decided not to, I probably forgot to revert this change. My bad.